### PR TITLE
Task 3.1-3.2 && Task 6 (FinalTask_rnn)

### PR DESCRIPTION
**Pull request by MaximKuznetsov17**
Создал свёрточную нейросеть, переобучил её до 100% на тренировочных данных
Затем во втором пунтке создавал свёрточные сети на Pytorch, в результате была достигнута точность 90% на тестовой выборке: 0.904318928149841

### CODE REVIEW

**Comment by MaximKuznetsov17** [comment]
Task 6
В ходе работы были выполнены три различных версии RNN, которые на тестовых данных давали точность:
LSTM: accuracy = 0.934
Bidirectional: accuracy = 0.953
Pretrained: accuracy = 0.951
